lib/SIL/SILGlobalVariable.cpp

```diff
@@ -129,6 +129,7 @@ static bool analyzeStaticInitializer(SILFunction *F, SILInstruction *&Val,
       if (I.getKind() != ValueKind::ReturnInst &&
```

Comment:
@bob-wilson Slava makes a good point. It should be pretty easy to add support for EnumInst where payloads are e.g. simple integers.